lib/zora/getZoraPfpLink.ts

```diff
@@ -0,0 +1,10 @@
+import { ZoraProfile } from "../zora.types"
+
+const getZoraPfpLink = (profile: ZoraProfile) => {
+    if (profile.avatar.startsWith("/")) {
+      return `https://zora.co/api/avatar/${profile.address}`
```

Comment:
If `profile.avatar` already starts with `/` wouldn't your current implementation lead to `avatar//url`?
- Why is the `//` included?